src/main/resources/org/jenkinsci/plugins/nomad/NomadSlaveTemplate/config.jelly

```diff
@@ -16,6 +16,10 @@
             <f:textbox/>
         </f:entry>
 
+        <f:entry title="${Constraints}" field="constraints" description="comma separated LTarget,RTarget,Operand">
```

Comment:
This currently will not work for distinct_host because I am automatically wrapping the LTarget with ${ } as Jenkins does not let you input these characters into a text field easily. I will change the code so LTarget is blank if not specified, this would allow distinct_host to work if you did something like:
Input: ,distinct_host,
However, this would not fix the distinct_property problem, unless I use an if statement to check for distinct_property, but that is not very future-proof